subgraph/core/src/entities/Round.ts

```diff
@@ -19,5 +20,7 @@ export function createRoundFromRoundInfo(
   round.repartitions = roundInfo.repartitions;
   round.penalties = roundInfo.pnkPenalties;
   round.dispute = disputeID.toString();
+  const courtID = contract.disputes(disputeID);
+  round.court = courtID.value0.toString();
```

Comment:
I think `courtID` here as variable name is a bit misleading, I would go with:
```suggestion
  const courtID = contract.disputes(disputeID).value0.toString();
  round.court = courtID;
```